decidim-proposals/spec/system/proposal_show_spec.rb

```diff
@@ -47,6 +47,24 @@ def visit_proposal
           end
         end
       end
+
+      describe "author tooltip" do
+        let(:user) { create(:user, :confirmed, organization: organization) }
+
+        before do
+          login_as user, scope: :user
+          visit current_path
+        end
+
+        context "when author doesn't restrict messaging" do
```

Comment:
When reading this the first time I thought author = author of message and was confused. Now it's obvious it's the proposal author, but first impressions, maybe being explicit is helpful: "proposal author" in the `describe` and `context` text.

Reply:
I love `rspec -fd --dry-run`: 

➜  decidim-proposals git:(feature/tooltip-author-msg-link) be rspec -fd --dry-run spec/system/proposal_show_spec.rb      

![image](https://user-images.githubusercontent.com/29486/106131071-7dc9d280-6162-11eb-9dcd-579661769357.png)